code/datums/abnormality/_ego_datum/zayin.dm

```diff
@@ -95,4 +95,18 @@
 
 /datum/ego_datum/armor/nightshade
 	item_path = /obj/item/clothing/suit/armor/ego_gear/nightshade
+// Pile of mail - Letter Opener
```

Comment:
cost is cut off here, needs to be fixed or nightshade will be broken by this PR.

Reply:
Also, keep the spacing between comments and code consistent with the rest of the file